### ISSUE TITLE
Error in prepareInputData DEMETER

### ISSUE THREAD

**Issue by AngelRuizMoreno** [opencobra/COBRA.tutorials#203]
Hi,

I have been running the DEMETER pipeline using the tutorial provided in cobratoolbox. However, I recently encountered this error when running line 9 of the tutorial.
```
[infoFilePath,inputDataFolder] = prepareInputData(infoFilePath,'spreadsheetFolder',spreadsheetFolder);
```

```
Unable to perform an assignment with 0 elements on the right-hand side.

Error in prepareInputData (line 137)
                infoFile{i,gramCol}=agoraInfoFile{taxRow,agoraGramCol};
```

I think some changes have been made in the core code of prepareInputData that now give this error.

I'm using Matlab 2020a and ibm_cplex as solver.

**Comment by almut-heinken** [comment]
Hello Angel,
I am not getting this error. May you ensure that the tutorials and papers
folder are up to date?
Best regards,
Almut

Angel J. Ruiz Moreno ***@***.***> schrieb am Di., 28. Nov.
2023, 11:13:

**Comment by AngelRuizMoreno** [comment]
I found the error. It was a problem on my end when working with a custom-made folder. I pulled the latest versions of tutorials and papers, and the tutorial runs fine. 

I also manage to perform the set-up for my custom analysis. 

Line 28 in the DEMETER tutorial file in the debugging section is still:
```
[debuggingReport, fixedModels, failedModels]=runDebuggingTools(refinedFolder,testResultsFolder,inputDataFolder,reconVersion,'numWorkers',numWorkers);
```
And rises an error. The line must be:

```
[debuggingReport, fixedModels, failedModels]=runDebuggingTools(refinedFolder,testResultsFolder,inputDataFolder,infoFilePath,reconVersion,'numWorkers',numWorkers);
```

I updated it in my own analysis, but the tutorial is not yet updated. Hopefully, this can be done soon as well.

Thank you. I will close this issue.
